SYCL/ESIMD/sycl_esimd_mix.cpp

```diff
@@ -10,9 +10,11 @@
 
 // REQUIRES: gpu
 // UNSUPPORTED: cuda || hip
+// TODO/FIXME: esimd_emulator support - timeout
+// XFAIL: esimd_emulator
```

Comment:
Changes in this test completely defeat the test's purpose. I think for now, we should just add `UNSUPPORTED: esimd_emulator`, and leave the rest as is.
Eventually, we might need to figure out a way to execute non-ESIMD kernels via IGC and ESIMD kernels - via esimd_emulator.